src/deploy_inst_tests.cc

```diff
@@ -138,6 +141,41 @@ TEST(DeployInstTests, NoDupProtos) {
   EXPECT_EQ(1, stmt->GetInt(0));
 }
 
+TEST(DeployInstTests, Latitude) {
+  std::string config = 
+     "<prototypes>  <val>foobar</val> </prototypes>"
+     "<build_times> <val>1</val>      </build_times>"
+     "<n_build>     <val>3</val>      </n_build>"
+     "<lifetimes>   <val>2</val>      </lifetimes>"
+     "<latitude>    2.0               </latitude>";
+
+  int simdur = 5;
+  cyclus::MockSim sim(cyclus::AgentSpec(":cycamore:DeployInst"), config, simdur);
+  sim.DummyProto("foobar");
+  int id = sim.Run();
+
+  QueryResult qr = sim.db().Query("AgentPosition", NULL);
+  EXPECT_EQ(qr.GetVal<double>("Latitude"), 2.0);
+}
+
+TEST(DeployInstTests, Longitude) {
+  std::string config = 
+     "<prototypes>  <val>foobar</val> </prototypes>"
+     "<build_times> <val>1</val>      </build_times>"
+     "<n_build>     <val>3</val>      </n_build>"
+     "<lifetimes>   <val>2</val>      </lifetimes>"
+     "<longitude>   2.0               </longitude>";
+
+  int simdur = 5;
+  cyclus::MockSim sim(cyclus::AgentSpec(":cycamore:DeployInst"), config, simdur);
+  sim.DummyProto("foobar");
+  int id = sim.Run();
+
+  QueryResult qr = sim.db().Query("AgentPosition", NULL);
+  EXPECT_EQ(qr.GetVal<double>("Longitude"), 2.0);
+}
+
```

Comment:
Should probably test latitude and longitude in the same test (corner case, checks for ordering issues, etc.)

Reply:
Thank you. 
I'll merge the test and add more.

Reply:
I've also noticed that I've never made sure to check if the input coordinates are within the correct range.

I'll update this as well.